src/Savegame/Soldier.h

```diff
@@ -78,10 +78,12 @@ class Soldier
 	SoldierDeath *_death;
 	SoldierDiary *_diary;
 	std::string _statString;
+	std::string _stateString;
```

Comment:
Please use different name, one `e` is very bad difference

Reply:
Yeah, I agree, what about tagString, extendedStatString, scriptedStatString, scriptString? What would you prefer?

Reply:
`_statTagString` to be similar to current one, but enogth distinct to do not make confusion like current one.